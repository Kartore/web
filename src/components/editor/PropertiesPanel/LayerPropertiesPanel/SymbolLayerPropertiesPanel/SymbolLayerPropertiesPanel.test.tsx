import { composeStories } from '@storybook/react';
import { render } from '@testing-library/react';
import { describe } from 'vitest';

import * as Stories from './SymbolLayerPropertiesPanel.stories.tsx';

const { Default } = composeStories(Stories);

describe('Component: SymbolLayerPropertiesPanel', () => {
  describe('Snapshot', () => {
    it('Default', () => {
      const { asFragment } = render(<Default />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
