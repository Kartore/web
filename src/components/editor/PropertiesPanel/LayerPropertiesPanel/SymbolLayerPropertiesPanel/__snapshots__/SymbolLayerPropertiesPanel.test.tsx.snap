// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Component: SymbolLayerPropertiesPanel > Snapshot > Default 1`] = `
<DocumentFragment>
  <div />
</DocumentFragment>
`;
